<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugWindow</class>
 <widget class="QDialog" name="DebugWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debug mode</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Select scan</string>
   </property>
  </widget>
  <widget class="QComboBox" name="scanSelector">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start scan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="configLoadButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Load config file</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="configFileDisplay">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::NoWrap</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="deviceDisplay">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>371</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Device type selected: none</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
